packages/react/src/radio-group/RadioGroup.tsx

```diff
@@ -0,0 +1,58 @@
+import * as RadixRadio from "@radix-ui/react-radio-group";
+import {
+  Children,
+  type ComponentPropsWithRef,
+  type ReactElement,
+  cloneElement,
+  forwardRef,
+  isValidElement,
+} from "react";
+
+import type { BoxProps } from "../box";
+
+import { Flex } from "../flex";
+import { RadioGroupItem } from "../radio-group-item";
+import { extractSprinkles } from "../sprinkles";
+
+type RadioGroupProps = BoxProps<
+  typeof RadixRadio.RadioGroup,
+  {
+    children:
+      | ReactElement<typeof RadioGroupItem>
+      | ReactElement<typeof RadioGroupItem>[];
```

Comment:
i don't think this is needed - we can just use
```suggestion
    children?: ReactNode;
```